ipyopenlayers/example.py

```diff
@@ -38,7 +48,27 @@ class Map(DOMWidget):
     value = Unicode('Hello World').tag(sync=True)
 
     layers = List(Instance(TileLayer)).tag(sync=True, **widget_serialization)
-
+    
     def add_layer(self, layer):
         # Copy layers (workaround ipywidgets issue)
         self.layers = self.layers + [layer]
+
+    def remove_layer(self, layer):
+        self.layers = [x for x in self.layers if x != layer]
+
+    def substitute(self, old, new):
```

Comment:
Suggestion for making the API more homogeneous:

```suggestion
    def substitute_layer(self, old, new):
```